src/include/sof/audio/dts/DtsSofInterfaceVersion.h

```diff
@@ -8,10 +8,10 @@
 
 #define DTS_SOF_INTERFACE_PRODUCT_NAME "DtsSofInterface"
 #define DTS_SOF_INTERFACE_PRODUCT_VERSION_MAJOR 1
-#define DTS_SOF_INTERFACE_PRODUCT_VERSION_MINOR 0
-#define DTS_SOF_INTERFACE_PRODUCT_VERSION_PATCH 7
+#define DTS_SOF_INTERFACE_PRODUCT_VERSION_MINOR 1
+#define DTS_SOF_INTERFACE_PRODUCT_VERSION_PATCH 3
 #define DTS_SOF_INTERFACE_PRODUCT_VERSION_BUILD 0
-#define DTS_SOF_INTERFACE_PRODUCT_VERSION_EXTRA "Dev"
+#define DTS_SOF_INTERFACE_PRODUCT_VERSION_EXTRA "RC2"
```

Comment:
Will RC2 be final for extra version ?

Reply:
We haven't found any other issue in current version. That's supposed to be.